static/js/typeahead_helper.js

```diff
@@ -100,10 +100,24 @@ exports.sorter = function (query, objs, get_item) {
    return results.matches.concat(results.rest);
 };
 
-exports.compare_by_pms = function (user_a, user_b) {
```

Comment:
I would recommend that you don't change this function. This way you can be sure that you haven't introduced any bug in `compare_by_pms`.